packages/react-centra-checkout/package.json

```diff
@@ -37,7 +37,8 @@
   },
   "dependencies": {
     "@noaignite/utils": "workspace:*",
-    "js-cookie": "^3.0.1"
+    "js-cookie": "^3.0.1",
+    "zod": "^3.23.8"
```

Comment:
Very unsure if we want to add `zod` as dependency in this package.
Considered more fine, if we can assure that it would be properly tree-shaken away from clientside usage of the package.

Reply:
This seems redundant. It is used in one place; I'd rather you create predicates for `bodyParserSchema` or if the intention is to use it package-wide; then I'd introduce Zod if it serves a more concrete purpose.